### PR TITLE
circuit-build@3.1.3

### PR DESCRIPTION
**Pull request by asanin-epfl**

### CODE REVIEW

File: var/spack/repos/builtin/packages/circuit-build/package.py
Comment (matz-e) [inline var/spack/repos/builtin/packages/circuit-build/package.py]: Is this going to turn into a non dev tag?

Comment (asanin-epfl) [reply var/spack/repos/builtin/packages/circuit-build/package.py]: Yes, I'm waiting for approval on this tag. This is just a test. Btw how to test PR's module now? I tried the usual:
```
  $ module purge
  $ unset MODULEPATH
  $ source /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/xxx/config/modules.sh
```
but received
```
-bash-4.2$ module purge
-bash-4.2$ unset MODULEPATH
-bash-4.2$ source /gpfs/bbp.cscs.ch/apps/hpc/jenkins/pulls/1029/config/modules.sh
-bash-4.2$ module load circuit-build
ModuleCmd_Load.c(213):ERROR:105: Unable to locate a modulefile for 'circuit-build'
```

Comment (matz-e) [reply var/spack/repos/builtin/packages/circuit-build/package.py]: `module load unstable circuit-build` should work, I hope!

Comment (asanin-epfl) [reply var/spack/repos/builtin/packages/circuit-build/package.py]: Yep, I tried but the old version `3.1.1` was loaded.
```
-bash-4.2$ module load unstable
-bash-4.2$ module load circuit-build/3.1.3
ModuleCmd_Load.c(213):ERROR:105: Unable to locate a modulefile for 'circuit-build/3.1.3'
```

Comment (matz-e) [reply var/spack/repos/builtin/packages/circuit-build/package.py]: I think there's a version pin in `deploy/environments/applications.yaml`. If you remove the version from `circuit-build` there, Jenkins should build the newest version automatically.

Comment (asanin-epfl) [reply var/spack/repos/builtin/packages/circuit-build/package.py]: thank you!

**Review by matz-e — DISMISSED** [review]
LGTM

**Review by matz-e — APPROVED** [review]

**Comment by asanin-epfl** [comment]
I had to merge `develop` because of the conflict with #1027

**Comment by asanin-epfl** [comment]
Thank you!

**Pull request merged**
